docs/Language Feature Status.md

```diff
@@ -8,10 +8,24 @@ efforts behind them.
 
 # C# Next
 
+| Feature | Branch | State | Developer | Reviewer | LDM Champ |
+| ------- | ------ | ----- | --------- | -------- | --------- |
+| [Relax ordering of `ref` and `partial` modifiers](https://github.com/dotnet/csharplang/issues/946) | [ref-partial](https://github.com/dotnet/roslyn/tree/features/ref-partial) | In Progress | [alrz](https://github.com/alrz) | [gafter](https://github.com/gafter) | [jcouv](https://github.com/jcouv) |
```

Comment:
Maybe remove this while you're at it. It's not being worked on. See [LDM-2021-07-12.md](https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-07-12.md)
The latest iteration lives at https://github.com/dotnet/roslyn/pull/42097